### ISSUE TITLE
90 degree rotation of beam position angle in restored image FITS header

### ISSUE THREAD

**Issue by rdeane** [ratt-ru/PyMORESANE#14]
Tried this with CASA and PyMORESANE and get different recorded BPA value in the restored image FITS header. Didn't really even need to check with CASA, I knew my beam orientation from the array I'm using (the EHT).

**Comment by o-smirnov** [comment]
I guess @JSKenyon is just using the wrong PA convention. Should be North through East.

**Comment by JSKenyon** [comment]
Yeah, that is most likely it - I had some trouble with fitting the primary beam - couldn't ever find a useful description of convention.

**Comment by rdeane** [comment]
Sorry - I could have have been more specific: I was referring to the synthesised beam position angle (i.e. the "BPA" FITS header keyword)
